x-pack/platform/plugins/shared/cases/public/components/markdown_editor/translations.ts

```diff
@@ -10,3 +10,17 @@ import { i18n } from '@kbn/i18n';
 export const PREVIEW = i18n.translate('xpack.cases.markdownEditor.preview', {
   defaultMessage: 'Preview',
 });
+
+export const NO_SIMULTANEOUS_UPLOADS_MESSAGE = i18n.translate(
```

Comment:
One again I didn't see this error in the UI.

Reply:
<img width="891" height="422" alt="image" src="https://github.com/user-attachments/assets/86f38b83-56dd-4fca-884d-1d79afd66aa0" />

If you try pasting multiple files (i.e. copy 2 image files from your desktop) and paste them here, this error message should show up.